### ISSUE TITLE
[Request] qadwaitadecorations and qadwaitadecorations-git

### ISSUE THREAD

**Issue by D3vil0p3r** [chaotic-aur/packages#2761]
### Package:

https://aur.archlinux.org/pkgbase/qadwaitadecorations
https://aur.archlinux.org/pkgbase/qadwaitadecorations-git

Depends:
https://aur.archlinux.org/pkgbase/qt5-wayland-decorations

### Purpose:

Qt decoration plugin implementing Adwaita-like client-side decorations

### Benefits:

Since qgnomeplatform-qt5 and qgnomeplatform-qt6 have been removed from Extra repository because no longer maintained, the proposed package should be their replacement for decorations: https://github.com/FedoraQt/QGnomePlatform/releases/tag/0.9.2

### Copyright:

LGPL

### Expected Interest:

Many

### Already available?

No

### Unique request?

Yes

### Banned package?

No

**Comment by coxackie** [comment]
Sorry for necro-bumping, but could you please also also add the non-git package `qadwaitadecorations` (includes qt5 and qt6 packages). It should also be accompanied by `qt5-wayland-decorations` (make and runtime dependency of the qt5 version)

**Comment by D3vil0p3r** [comment]
> Sorry for necro-bumping, but could you please also also add the non-git package [`qadwaitadecorations`](https://aur.archlinux.org/pkgbase/qadwaitadecorations) (includes qt5 and qt6 packages). It should also be accompanied by `qt5-wayland-decorations` (make and runtime dependency of the qt5 version)

Open a new request please.

**Comment by coxackie** [comment]
@D3vil0p3r  @xiota I guess reopening this means I do not have to open a new request. Many thanks!

**Comment by xiota** [comment]
@coxackie It's fine for closely related packages, like -git and non-git, to go together, as long as there's a actual need or desire for both.

**Comment by coxackie** [comment]
@xiota thanks, in this case the desire (and, maybe, need) is that the -git met package does not have the qt5 version. Maybe the author of the git package can now include it, given that the wayland-decoration patches for qt5 have been included.

**Comment by xiota** [comment]
@coxackie You can write a comment at [aur/qadwaitadecorations-git](https://aur.archlinux.org/pkgbase/qadwaitadecorations-git) to request it.

**Comment by coxackie** [comment]
somehow, `qadwaitadecorations-qt{5,6}` is at v0.1.2-2, while in the AUR it is at v0.1.3-1 since 2023-10-16.

**Comment by xiota** [comment]
@dr460nf1r3 Seems like packages after about ufscar-hpc/hourly.2.txt line 1731 aren't being rebuilt?  Some packages before that line are also skipped.  Any idea why packages are being skipped?

**Comment by dr460nf1r3** [comment]
Interesting finding. Possibly a stuck build blocking the routine?

**Comment by xiota** [comment]
@coxackie Is there urgency updating the package?  If not, leaving it at the old version for now would help identification of the reason packages are being skipped during build.

**Comment by xiota** [comment]
@dr460nf1r3 Seems like troublesome packages are missing a newline at the end of the PKGBUILD, so the attempted fix in toolbox doesn't seem to be working.

**Comment by coxackie** [comment]
@xiota no problem for me to wait until you figure this out, although from what I understand it may be beyond you...

**Comment by hsnfirdaus** [comment]
The old version in chaotic AUR (v0.1.2-2) have bug in qt6 version in my device. It will make qt6 application crash, but it fixed in 0.1.3-2 version.

**Comment by xiota** [comment]
`qadwaitadecorations` has been rebuilt.

`qadwaitadecorations-git` has build errors.

**Comment by coxackie** [comment]
I think `qadwaitadecorations-qt6` may need rebuilding against the new `qt6-wayland` package.

**Comment by dr460nf1r3** [comment]
That is highly likely. Doing!

**Comment by coxackie** [comment]
thanks @dr460nf1r3 - it has not appeared yet at the repo, hopefully all is ok...

**Comment by coxackie** [comment]
pardon for the nudge, is there any updates on this? Many thanks in advance!

**Comment by xiota** [comment]
Are none of these what you want?
```
$ pacman -Ss qadwaita
chaotic-aur/qadwaitadecorations-qt5 0.1.5-1
    Qt5 decoration plugin implementing Adwaita-like client-side decorations
chaotic-aur/qadwaitadecorations-qt5-git 0.1.5.r0.g8f7357c-1
    Qt5 decoration plugin implementing Adwaita-like client-side decorations
chaotic-aur/qadwaitadecorations-qt6 0.1.5-1
    Qt6 decoration plugin implementing Adwaita-like client-side decorations
chaotic-aur/qadwaitadecorations-qt6-git 0.1.5.r0.g8f7357c-1
    Qt6 decoration plugin implementing Adwaita-like client-side decorations
chaotic-aur/qt5-wayland-decorations 5.15.13+kde+r60-1
    Provides APIs for Wayland (with patch required for shadows on qadwaitadecorations-qt5)
```

**Comment by coxackie** [comment]
> Sorry... misunderstood... they need to be rebuilt? Will bump now.

yes, thanks. They are not compatible with the current qt-wayland packages, they throw errors.

**Comment by xiota** [comment]
They're rebuilt now:
```bash
$ pacman -Si qadwaitadecorations-qt6 | grep Date
Build Date      : Sun 21 Apr 2024 06:17:11 PM UTC

$ pacman -Si qadwaitadecorations-qt6-git | grep Date
Build Date      : Sun 21 Apr 2024 06:17:44 PM UTC
```

**Comment by xiota** [comment]
I'm switching these to custom packages:

* Fix for Qt 6.10.
* Drop -qt5 component.

Metrics:

* qadwaitadecorations-qt5 # 9
* qadwaitadecorations-qt5-git # 33
* qadwaitadecorations-qt6 # 29
* qadwaitadecorations-qt6-git # 38
